### DISCUSSION TITLE
oidc_util_http_call: curl_easy_perform() failed on:

### DISCUSSION THREAD

**Discussion by AuthNovice** [OpenIDC/mod_auth_openidc#905] [Q&A]
Our OAuth Authentication has just stopped working.  The only errors in the Apache logs are as below.

[Wed Aug 17 10:11:41.009230 2022] [auth_openidc:error] [pid 3968:tid 9052] [client xxx.xx.xx.xxx:xxxxx] oidc_util_http_call: curl_easy_perform() failed on: https:///token (), referer: https://
[Wed Aug 17 10:11:41.009230 2022] [auth_openidc:warn] [pid 3968:tid 9052] [client xxx.xx.xx.xxx:xxxxx] oidc_proto_token_endpoint_request: error when calling the token endpoint (https:///token), referer: https:///
[Wed Aug 17 10:11:41.009230 2022] [auth_openidc:error] [pid 3968:tid 9052] [client xxx.xx.xx.xxx:xxxxx] oidc_proto_resolve_code_and_validate_response: failed to resolve the code, referer: https:///

I'm using mod_auth_openidc on Windows 2016 server using Apache 2.4.34.

Is there any advise please on where to look to resolve this?  This has been working successfully since 2019.

**Comment by zandbelt** [comment]
support for the Windows build of mod_auth_openidc is only available under a commercial agreement via sales@zmartzone.eu
